src/android/unregisterNativeModule.js

```diff
@@ -60,15 +62,7 @@ module.exports = function unregisterNativeAndroidModule(name, dependencyConfig,
     readFile(projectConfig.mainActivityPath)
   );
 
-  /**
-   * Check if module has been installed already
-   */
-  const isInstalled = compose(
-    (content) => ~content.indexOf(getAddPackagePatch(dependencyConfig)),
```

Comment:
For some reason that was broken. Since `link` uses different check that works, I decided to use it instead (`isInstalled` method was extracted as we discussed on Skype)